VMD_scripts/blobulation.tcl

```diff
@@ -222,12 +222,22 @@ proc hydropathyScores { hydropathyList Sequence } {
 	set hydroScored {}
 	foreach amino $Sequence {
 		if {[lsearch -exact $hydropathyList $amino] == -1} {
-			puts "Unkown amino acid ending program"
-			return -1
-		}
-		
+			if {$amino == "HID" || $amino == "HIE"} {
+				set value [dict get $hydropathyList "HIS"]
+			} else {
+				set aminoList {}
```

Comment:
aminoList is the list of unrecognized amino acids, right? Can you rename this to something like unknownResnames?

Reply:
Done